cmd/harmony/default.go

```diff
@@ -178,6 +178,7 @@ var defaultStagedSyncConfig = harmonyconfig.StagedSyncConfig{
 	MaxMemSyncCycleSize:    1024,  // max number of blocks to use a single transaction for staged sync
 	UseMemDB:               true,  // it uses memory by default. set it to false to use disk
```

Comment:
is there a limit for `UseMemDB` on the memory usage to avoid a possible OOM ?

Reply:
Exactly, That's the reason I added a parameter for it. It depends on the available memory and system resources. I used the small Cycle Size to make sure it works on almost every node with minimum required spec.

Reply:
what's the parameter and can this be somewhat dynamic based on the machine memory ?

Reply:
it can be enabled on every single node regardless of the specs. But MaxMemSyncCycleSize should be set appropriately.

Reply:
so it's manual. Do you have a recommended settings that match the expected memory for `MaxMemSyncCycleSize`

Reply:
The default value of MaxMemSyncCycleSize is the recommended one. That's the same value used by erigon as well. It is on 1024.